### PR TITLE
Trajectory Optimization with Thrusters and Planar SRB

### PR DESCRIPTION
**Pull request by vincekurtz**
Adds an example of doing trajectory optimization with a body-fixed thruster for the planar SRB model.

Pros and cons of modeling the thrusters in this way:
- We can treat a body-fixed thruster just like any other actuated joint.
- This accounts for the fact that thrust is not applied exactly at the CoM.
- Extending this to 3D is possible but non-trivial. 
- Extending this to use thrust vectoring is not possible (I think).
- This introduces a singularity: the torso cannot rotate more than 90 degrees. That may be fine for the proof-of-concept tasks we're considering now, but won't work for backflips.

### CODE REVIEW

File: models/urdf/planar_srb_with_thruster.urdf [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+        </inertial>
+        <collision>
+            <origin rpy="0 0 0" xyz="0 0 -50"/>
+            <geometry>
+                <box size="100 100 100"/>
+            </geometry>
+        </collision>
+    </link>
+
+    <!-- *************************** JOINTS *************************** -->
```
Comment (sesteban951) [inline models/urdf/planar_srb_with_thruster.urdf:182]: joints are good, but you missed one

File: optimize_planar_srb_with_thruster.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+##
+
+import numpy as np
+import time
+
+from pydrake.all import (StartMeshcat, DiagramBuilder, AddMultibodyPlantSceneGraph,
+                         AddDefaultVisualization, Parser, MultibodyPlantConfig,
+                         AddMultibodyPlant, ConstantVectorSource, Simulator,
+                         LeafSystem, BasicVector, ValueProducer, AbstractValue,
+                         JointActuatorIndex, PdControllerGains)
```
Comment (sesteban951) [inline optimize_planar_srb_with_thruster.py:18]: too much imported

**Review by sesteban951 — CHANGES_REQUESTED** [review]
good, small details

File: optimize_planar_srb_with_thruster.py (hunk 2 above)
Comment (vincekurtz) [reply optimize_planar_srb_with_thruster.py:18]: oops, my bad

**Pull request merged**